### PR TITLE
Fix/make compatible for xcode10.3

### PR DESCRIPTION
**Pull request by mwakizaka**
# Objectives
1. Bump up Swift version to 5.0
2. Add back button
3. Localization(Japanese/English)
    - Note that when you choose other than Japanese/English 

# Japanese
IMG_5571
---
IMG_5570

# English
IMG_5572
---
IMG_5573

### CODE REVIEW

File: magic_pod_demo_app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -91,12 +100,12 @@
...
 			attributes = {
 				LastSwiftUpdateCheck = 0730;
-				LastUpgradeCheck = 0730;
+				LastUpgradeCheck = 1030;
 				ORGANIZATIONNAME = TRIDENT;
 				TargetAttributes = {
 					08A5C0421D3CAE0F00D8DD31 = {
 						CreatedOnToolsVersion = 7.3;
-						DevelopmentTeam = F2R6DW797Q;
+						DevelopmentTeam = 9U4PT397Q4;
```
Comment (mwakizaka) [inline magic_pod_demo_app.xcodeproj/project.pbxproj]: Is it better to revert this change?

Comment (NozomiIto) [reply magic_pod_demo_app.xcodeproj/project.pbxproj]: One solution: https://stackoverflow.com/questions/39669661/how-to-prevent-xcode-8-from-saving-development-team-in-pbxproj
